courier/template/testhelpers/testhelpers.go

```diff
@@ -26,7 +26,7 @@ import (
 )
```

Comment:
These were running for around 5s and are now < 0.5s total.

(They are called by the `github.com/ory/kratos/courier/template/email` tests, which were running for around 40s and are now at 6.5s)